### PR TITLE
Rewrite Apache SNMP Agent to Python3

### PR DESCRIPTION
**Pull request by hanserasmus**
Thanks to @murrant this file is now python3 compatible.

### CODE REVIEW

File: snmp/apache-stats.py [hunk 1]
```diff
@@ -18,72 +18,68 @@
 #
 #
 
-CACHETIME = 30
-CACHEFILE = '/var/cache/librenms/apache-snmp'
+CACHETIME=30
```
Comment (SourceDoctor) [inline snmp/apache-stats.py:21]: it's python not bash
please do space between operators
-> PEP8 compatibilty

File: snmp/apache-stats.py [hunk 2]
```diff
@@ -18,72 +18,68 @@
...
-if os.path.isfile ( CACHEFILE ) \
-	and ( time.time() - os.stat ( CACHEFILE )[8] ) < CACHETIME:
-	# use cached data
-	f = open ( CACHEFILE, 'r' )
-	data = f.read()
-	f.close()
+if os.path.isfile(CACHEFILE) \
+        and(time.time() - os.stat(CACHEFILE)[8]) < CACHETIME:
+        # use cached data
+        f=open(CACHEFILE, 'r')
```
Comment (SourceDoctor) [inline snmp/apache-stats.py:30]: also here

File: snmp/apache-stats.py [hunk 3]
```diff
@@ -18,72 +18,68 @@
...
-	from urlgrabber import urlread
-	data = urlread ( 'http://localhost/server-status?auto',
-					user_agent = 'SNMP Apache Stats' )
-	# write file
-	f = open ( CACHEFILE+'.TMP.'+`os.getpid()`, 'w' )
-	f.write ( data )
-	f.close()
-	os.rename ( CACHEFILE+'.TMP.'+`os.getpid()`, CACHEFILE )
+        # Grab the status URL (fresh data), needs package python-urlgrabber
+        from urlgrabber import urlread
```
Comment (SourceDoctor) [inline snmp/apache-stats.py:35]: imports should be alway on top of the file

File: snmp/apache-stats.py [hunk 4]
```diff
@@ -18,72 +18,68 @@
...
 ]
 for param in dataorder:
-	try:
-		print params[param]
-#                print param
-	except:	# not all Apache's have all stats
-		print 'U'
+    try:
+        print(params[param])
+    except: # not all Apache's have all stats
```
Comment (SourceDoctor) [inline snmp/apache-stats.py:80]: i think  `except KeyError:`  is better to catch only not existing key - errors

File: snmp/apache-stats.py (hunk 1 above)
Comment (hanserasmus) [reply snmp/apache-stats.py:21]: Sorry, will do.

File: snmp/apache-stats.py (hunk 4 above)
Comment (hanserasmus) [reply snmp/apache-stats.py]: You mean change:
`except:  # not....`

to
`except KeyError:  # not...`
?

**Pull request merged**